validation-test/SIL/verify_all_overlays.sil

```diff
@@ -1,6 +1,11 @@
-// RUN: for x in %platform-sdk-overlay-dir/*.swiftmodule; do [[ $(basename "$x") = Swift.swiftmodule ]] && continue; llvm-bcanalyzer $x | %FileCheck %s; %target-sil-opt -sdk %sdk -enable-sil-verify-all $x > /dev/null; done
+// RUN: %empty-directory(%t)
+// RUN: for x in %platform-sdk-overlay-dir/*.swiftmodule; do [[ $(basename "$x") = Swift.swiftmodule ]] && continue; llvm-bcanalyzer $x | %FileCheck %s || echo "$x (bcanalyzer)" >> %t.txt; %target-sil-opt -sdk %sdk -enable-sil-verify-all $x > /dev/null || echo "$x (sil-opt)" >> %t/failures.txt; done
```

Comment:
Oof, the bcanalyzer one is also supposed to cat into `%t/failures.txt`. I'll fix that afterwards; that part isn't failing.